pkg/collector/corechecks/system/windowscertificate/windows_certificate.go

```diff
@@ -608,10 +608,18 @@ func findCertificatesInStore(storeHandle windows.Handle, subjectFilters []string
 				}
 			}
 
+			// >>> ADD: populate thumbprint on the filtered path <<<
+			certThumbprint, err := getCertThumbprint(certContext)
+			if err != nil {
+				log.Errorf("Error getting certificate thumbprint: %v", err)
+				// keep or 'continue' depending on desired behavior; keeping matches enumerate path semantics only if err is rare
```

Comment:
We should set continue here. We shouldn't try to add keep this certificate if we fail to retrieve its thumbprint.

```suggestion
				continue
```